pkg/acme/cert_request.go

```diff
@@ -11,16 +11,40 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strings"
 	"sync"
 
 	"time"
 
 	"github.com/cenk/backoff"
 	"github.com/jetstack/kube-lego/pkg/kubelego_const"
+	"github.com/prometheus/client_golang/prometheus"
 	"golang.org/x/crypto/acme"
 	"golang.org/x/net/context"
 )
 
+var (
+	certReqs = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "cert_reqs_total",
+			Help: "Number of ACME certificate requests",
+		},
+		[]string{"domains"},
+	)
+	certReqErrors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "cert_errors_total",
+			Help: "Number of ACME certificate request errors",
+		},
+		[]string{"domains"},
+	)
+)
+
+func init() {
+	prometheus.MustRegister(certReqs)
+	prometheus.MustRegister(certReqErrors)
```

Comment:
Please get rid of the `init()` and place that into kube-lego's https://github.com/jetstack/kube-lego/blob/master/pkg/kubelego/kubelego.go#L131